backends/bmv2/run-bmv2-test.py

```diff
@@ -137,8 +137,6 @@
 sys.path.append(str(ROOT_DIR))
 
 from bmv2stf import Options, RunBMV2
-from scapy.layers.all import *
-from scapy.utils import *
```

Comment:
I have tried to find any mentions of Scapy names from this file, but have not found any.  Unfortunately it is difficult to be 100% certain with `from <module> import *`, because they could be anywhere in the file without having the module name in front of them.

In the git commit history for run-bmv2-test.py, I found that back in 2016 there were some uses of `rdpcap` and `Ether` symbols from scapy in this file, but al of those were removed by 2016 or 2017, thus it appears that these import statements are vestigial.